.github/workflows/run-presubmit-checks.yml

```diff
@@ -44,3 +44,41 @@ jobs:
     - working-directory: Frontend/ui-library
       run: npm ci && npm link ../library && npm run build && npm run lint
 
+  run-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout source code"
+        uses: actions/checkout@v3
+      - working-directory: SignallingWebServer
+        run: |
+          ./platform_scripts/bash/start.sh --rest_api &
+          curl --retry 10 --retry-delay 20 --retry-connrefused http://localhost/api/status
+
+  run-windows:
+    runs-on: windows-latest
+    steps:
+      - name: "Checkout source code"
+        uses: actions/checkout@v3
+      - working-directory: SignallingWebServer
+        run: |
```

Comment:
Can we make the step more descriptive like: `- name: "Build, start, and query signalling server"`?

Reply:
Same with above for Linux